setup.py

```diff
@@ -20,7 +20,8 @@
             "myst-parser",
             "sphinx-book-theme",
             "sphinx-panels",
-            # pinned because of this bug https://github.com/pydata/pydata-sphinx-theme/pull/509
+            "sphinx-design~=0.0.11",
+# pinned because of this bug https://github.com/pydata/pydata-sphinx-theme/pull/509
```

Comment:
careful, this changed the meaning of the comment

Reply:
my bad, fixed in 72c5bf10bca96bead31321c70a1cb66377267708